tests/test_flow/test_persistence_compatability.py

```diff
@@ -0,0 +1,25 @@
+import pytest
+
+from .generate_test_compatability_cache import Harness
+
+
+@pytest.fixture
+def older_serialized_cache_harness(builder, datadir):
+    harness = Harness()
+    harness.builder.set('core__persistent_cache__flow_dir', datadir)
+    return harness
+
+
+# to regenerate cache, run
```

Comment:
I think we might want a bit more text here explaining what this test is doing and why someone would or wouldn't want to regenerate the cache.

Also, super super nit, but just for consistency, could we use capitalization and periods in comments?